lib/chart/base.dart

```diff
@@ -535,6 +541,9 @@ class ChartBloc extends WindowBloc<ChartState> {
   /// Open the [SeriesEditor] dialog to edit a series.
   Future<void> _editSeries(BuildContext context, SeriesInfo series) async {
     WorkspaceViewerState workspace = WorkspaceViewer.of(context);
+    if (workspace.info?.instrument?.schema == null) {
+      throw ArgumentError("The chosen instrument has no schema");
+    }
```

Comment:
Should this be a `reportError` too?

Reply:
It doesn't need to be- all exceptions are handled and displayed via `reportError` in `main.dart`